seroanalytics/epikinetics
# shiny app for diagnostics

This PR adds a function `biokinetics$inspect` which opens a shiny app and allows the user to inspect data and priors before fitting. Will add the post-fitting diagnostic plots in a future PR.

It also adds a vignette detailing the input data plots available. Again, will add info about output plots in the next PR.

To test out the shiny app:

```
devtools::load_all()
data <- data.table::fread(system.file("delta_full.rds", package = "epikinetics"))
mod <- biokinetics$new(data = data, covariate_formula = ~0 + infection_history)
mod$inspect()
```

I haven't really attempted to write tests for the shiny logic. It's quite a simple app and the tests are a massive pain to write and run, so I don't think the time involved is worth it right now. So would be very useful if you could have a play around with it manually to confirm working as expected!
